subworkflows/local/fusioninspector_workflow.nf

```diff
@@ -17,19 +17,18 @@ workflow FUSIONINSPECTOR_WORKFLOW {
     main:
         ch_versions = Channel.empty()
         index ="${params.starfusion_ref}"
-        ch_fusion_list = params.fusioninspector_filter ? fusion_list_filtered : fusion_list
 
         if (params.whitelist)  {
-            ch_whitelist = ch_fusion_list.combine(Channel.value(file(params.whitelist, checkIfExists:true)))
+            ch_whitelist = fusion_list_filtered.combine(Channel.value(file(params.whitelist, checkIfExists:true)))
```

Comment:
I wonder about the naming convention, sometimes channels seem to start by `ch` sometimes they don't. Should this be `ch_fusion_list_filtered` ?

Reply:
So I left it like this for the moment and opened: https://github.com/nf-core/rnafusion/issues/392 to harmonize in one go